Examples/Physics_applications/capacitive_discharge/PICMI_inputs_1d.py

```diff
@@ -108,7 +108,6 @@ def solve(self):
         calculating phi from rho."""
 
         left_voltage = 0.0
-        t = self.sim_ext.gett_new()
         right_voltage = eval(self.right_voltage)
```

Comment:
This line should be something like
```suggestion
        right_voltage = eval(self.right_voltage, {'t':sim.extension.warpx.gett_new(0)})
```
with the line above it removed since `self.right_voltage` depends on `t`. 
(If the CI test does not crash I'm going to be very confused 😃 )